securitytxt-signer.sh

```diff
@@ -82,7 +86,7 @@ else
       | grep "sec" \
       | grep -Eo 'expires:\ [0-9\-]+' \
       | awk '{ print $2}' \
-      | date -Iseconds -u -f - \
+      | ( [[ IS_MAC ]] && date -Iseconds -u || date -Iseconds -u -f - ) \
```

Comment:
The `[[ IS_MAC ]]` is always `true`.

```suggestion
      | ( [[ "$IS_MAC" = true ]] && date -Iseconds -u || date -Iseconds -u -f - ) \
```

See:

```
$ IS_MAC=true ; echo "foo" | ([[ $IS_MAC ]] && sed 's/foo/true/' || sed 's/foo/false/')
true
$ IS_MAC=false ; echo "foo" | ([[ $IS_MAC ]] && sed 's/foo/true/' || sed 's/foo/false/')
true

$ IS_MAC=true ; echo "foo" | ([[ "$IS_MAC" = true ]] && sed 's/foo/true/' || sed 's/foo/false/')
true
$ IS_MAC=false ; echo "foo" | ([[ "$IS_MAC" = true ]] && sed 's/foo/true/' || sed 's/foo/false/')
false
```

I think the same mistake elsewhere causes the `Expires:` field being empty on Linux.

Reply:
Updated suggestion per the other comment with `1` & `0`:
```suggestion
      | ( (( IS_MAC )) && date -Iseconds -u || date -Iseconds -u -f - ) \
```

Reply:
On the other hand we should consider [SC2015](https://www.shellcheck.net/wiki/SC2015):

> Note that `A && B || C` is not if-then-else. C may run when A is true.

This might require a full if-then-else:

```bash
  if (( IS_MAC )); then
    KEY_EXPIRES=$(
      echo "$KEY_INFO" \
        | grep "sec" \
        | grep -Eo 'expires:\ [0-9\-]+' \
        | awk '{ print $2}' \
        | date -Iseconds -u \
        | sed -e 's/+00:00$/Z/'
      )
  else
    KEY_EXPIRES=$(
      echo "$KEY_INFO" \
        | grep "sec" \
        | grep -Eo 'expires:\ [0-9\-]+' \
        | awk '{ print $2}' \
        | date -Iseconds -u -f - \
        | sed -e 's/+00:00$/Z/'
      )
  fi
```

Reply:
As I have fixed the ShellCheck pipeline in `master` it now correctly fails.